src/utils/openAiModels.ts

```diff
@@ -0,0 +1,12 @@
+const models = [
+  'gpt-3.5-turbo',
+  'text-davinci-003',
```

Comment:
davinci wont work as it's not the same openAI API endpoint, we use `CreateChatCompletion` for 3.5 and 4, but `CreateCompletion` for < 3.5

Reply:
let's remove it, 3.5 and 4 is enough

Reply:
Went ahead and removed the other model leaving gpt-3.5-turbo and gpt-4 as the only valid options